### DISCUSSION TITLE
RootPainter0.2.27 Numpy issue

### DISCUSSION THREAD

**Discussion by beechwood13** [Abe404/root_painter#132] [General]
Hi,
I have the following problem:
System:
Ubuntu 23.10
NVIDIA GA102GL [RTX A5000] with 64GB video RAM
Python3.9 form Miniconda
RootPainter 0.2.27 is installed from the tar.gz archive (both server and client on the same machine)
Compiling and setup went smoothly and everything seemed to work well. 
However, when trying to extract lenghtes for RVE, I get the following error:
-----------------------------------------------
Seconds to get metrics:  0.08
Seconds to get metrics:  0.07
log_exception 2024-02-19 22:14:10.305399|sys_exception|Error. type: <class 'AttributeError'>, value: module 'numpy' has no attribute 'int'.
`np.int` was a deprecated alias for the builtin `int`. To avoid this error in existing code, use `int` by itself. Doing this will not modify any behavior and is safe. When replacing `np.int`, you may wish to use e.g. `np.int64` or `np.int32` to specify the precision. If you wish to review your current use, check the release note link for additional information.
The aliases was originally deprecated in NumPy 1.20; for more details and guidance see the original release note at:
    https://numpy.org/devdocs/release/1.20.0-notes.html#deprecations, traceback:   File "base_extract.py", line 50, in run
  File "extract_length.py", line 31, in save_length_to_csv
  File "numpy/__init__.py", line 305, in __getattr__
------------------------------------------------------
(I remember I had a similar Numpy issue earlier in a different environment)
Thanks for helping in advance!

**Comment by Abe404** [comment]
Thanks for making me aware of this. The newer version of numpy don't support np.int and requires me to just use the built in int instead.

I've just made this change on the master branch: https://github.com/Abe404/root_painter/commit/fd3ce9f7ab255eff627bc3a99ed4462b1ceaddc2

Would you be able to run your trainer from source by cloning the master branch?

If so, then this should hopefully now be fixed for you. Otherwise I can create a new release tomorrow.

**Comment by Abe404** [comment]
BTW for RVE, you want to generate segmentations in RVE compatible format. I'm not sure why you would generate length from RootPainter for use in RVE.
